src/bun.js/bindings/webcore/FetchHeaders.cpp

```diff
@@ -261,30 +262,58 @@ void FetchHeaders::filterAndFill(const HTTPHeaderMap& headers, Guard guard)
     }
 }
 
-static NeverDestroyed<const String> setCookieLowercaseString(MAKE_STATIC_STRING_IMPL("set-cookie"));
+static const WTF::String setCookieLowercaseString = WTF::staticHeaderNames[static_cast<uint8_t>(HTTPHeaderName::SetCookie)];
```

Comment:
can we move this to be inside the usage so it doesn't impact startup time?

Reply:
updated